quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/MappedTripleGroupServiceImpl.java

```diff
@@ -0,0 +1,153 @@
+package edu.asu.diging.quadriga.core.service.impl;
+
+import org.bson.types.ObjectId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.quadriga.core.data.MappedTripleGroupRepository;
+import edu.asu.diging.quadriga.core.exceptions.CollectionNotFoundException;
+import edu.asu.diging.quadriga.core.exceptions.InvalidObjectIdException;
+import edu.asu.diging.quadriga.core.exceptions.MappedTripleGroupNotFoundException;
+import edu.asu.diging.quadriga.core.model.MappedTripleGroup;
+import edu.asu.diging.quadriga.core.model.MappedTripleType;
+import edu.asu.diging.quadriga.core.service.CollectionManager;
+import edu.asu.diging.quadriga.core.service.MappedTripleGroupService;
+
+@Service
+public class MappedTripleGroupServiceImpl implements MappedTripleGroupService {
```

Comment:
since all the methods in this class are about mapped triple groups, the method names do not need to contain that. E.g. `updateMappedTripleGroupNameById` => `updateName`